sara_admitance:
    joint_names:
        - right_shoulder_roll_joint
        - right_shoulder_pitch_joint
        - right_shoulder_yaw_joint
        - right_elbow_pitch_joint
        - right_elbow_yaw_joint

    # Here are the link names
    right_shoulder_roll_joint:
        - right_clavicular_link

    right_shoulder_pitch_joint:    
        - right_upper_arm_upper_link

    right_shoulder_yaw_joint:    
        - right_upper_arm_lower_link

    right_elbow_pitch_joint:    
        - right_forearm_upper_link

    right_elbow_yaw_joint:    
        - right_wrist_upper_link